embulk-input-mysql/src/main/java/org/embulk/input/mysql/MySQLInputConnection.java

```diff
@@ -59,4 +62,31 @@ public TimeZone getServerTimezoneTZ()
     {
         return ((ConnectionImpl) connection).getServerTimezoneTZ();
     }
+
+    @Override
+    public void before_load()
+        throws SQLException
+    {
+        // TODO error check.
+        TimeZone svr_tz = MySQLTimeZoneBuilder.fromSystemTimeZone(connection);
+
+        String usr_tz_name = System.getProperty("user.timezone");
+        TimeZone usr_tz = TimeZone.getTimeZone(usr_tz_name);
+
+        //
+        // Compare offset only. Although I expect to return true, the following code return false,
+        //
+        // TimeZone tz_jst   = TimeZone.getTimeZone("JST");
+        // TimeZone tz_gmt9  = TimeZone.getTimeZone("GMT+9");
+        // tz_jst.hasSameRules(tz_gmt9) // return false.
+        //
+        if( svr_tz.getRawOffset() != usr_tz.getRawOffset() ) {
+            logger.warn(String.format(Locale.ENGLISH,
+                    "The server timezone offset(%s) and client timezone(%s) has different timezone offset. The plugin will fetch wrong datetime values.",svr_tz.getID(),usr_tz_name));
```

Comment:
How about following message?
`The client timezone(%s) is different from the server timezone(%s). ...`